Integrations/integration-Box.yml

```diff
@@ -6,193 +6,327 @@ display: Box
DeAw6xbmpiqOnsFXuufhvNiDgCbRDoo7CPA9wX4sHULEQ2bArg1Fg+/3NM+uc86hmg0Mvm+dlW5DMDbrFuIaNheEuAb45cnf3z1LClZxzSLTFD8tG5YQU3V95IqDgq81J+tQ4jqwfTeF94Sj0oPAdjDuqUZCTA/56amDff1fAS4SeTd1GN7L0xOAeRkQArWPUS0SX+LIEf4buoEDv+oEeTSbXO32tLZC4pvg4dVEdU6BXBLLB7unnNTl3P4V11OKXYLgGXWHU1IBXIah39EwzenY8IrEiIL4CXrlmYUqXPJ5ryeKwCbULZ7+dZDY4bOEcXZALaw7iGif/OiAN9ySqt+0tO5+6B1DFElZHLL3qax6AKoHgX+LkJUY+ThCDgr7yYfsS5pZl7Qdwog11p3NBMVXBCkU1+z7iCqR16ueAQcnQMuMqsaAXpzbiq9mV9DzcrLvZCEU+oCcAqAmHEOUbN7SUW/MxSTH8xtT621jiGqBtcv7OY4OE8VWesWov/X3r0Hx1mddxz/PefdlbGNi2UsS1rZwSYhBHwBB0LAQG0CwZhYlnblFRDqxlwCnVygaUPSyUxjpS7T0EKAdIZLyhgayBCv0K4uwWAImITaEO7IGBeIWcD2rizZFsZXafc9T/8QbZhCjCytdPby+/zpmX3f72g0s6+e9/gcwhYR/V5LfU2H6xACoCqR1vSzAM5wnVIaZHW8vuoy7nFJNHSRROpvAdzquqMkCA55xpzcvKQqeWQfo5K3tHX7PKtyK/iAQeSCD5VfQv0fxxumbnMdQ+RCQ2v6YlW9Cdw/hmjUCdCrgpszHm7jC6j8Eo6n54voU647ip5iU9YLntVeV7HXdQpRQRt4cfEAgK+7TikBt8TDoe8f6Yc4AKQBqhJpT9XCygoAX3SdQ1Qi1vhi/qGtvmqj6xAi16Ix9Wwg9Q2F/BMENa57iErAXgC3WfT9rDU8433XMfTJwonU/QL8leuOIrbbwjujNVy5xXUIUTGIxraOtUHvaQVOc91SxN7zMkfNaW6cdMT7LnIASB8TjqfOgWClAAtctxAVIRXBQ1mYlRz8EX3cNXdrsKey6zJRXQGeGEw0Evao6L/4UnYHVzzlv2Vru8bvO2BfFOBE1y1FqB9WvxJvqFnvOoSomERj3VV+MPs8gKmuW4pQBlbOjjdUPz+UD3MASH8WB4FEuSWCZmvtjYnI1FddtxDlu2hsU5kNli9X4B/BB0iiXDggKv9u4f9bIjJ1l+sYGrz6ROo0A2wAUOa6pZio6JWJ+pp7XXcQFaOGtu1z1cp/ARjnuqW4yI/j4eqVQ/50LlOoOHEQSDQ8Av2NWu/GeEPVs65biArNnwaBsgLQkOseooIjOCRWbjdZ3NLcWN3jOoeGJtKa+hEUN7ruKCJ3xsOhb7mOICpm4XjqchE84LqjaCjWzemsvqCpSexQL8EBIA2OqjS0pusArFDgVNc5RAVB8XsjuPGhcOgx1ylEhW5JW8+EgM18B8D1ACpd9xAVgAwUD4r1/7ll6bS3XMfQ8CxYp4FJe9KtUHzNdUsR2HBwzP6vPHLxCX2uQ4iKXUNr6nZVXOe6o/BJygvoWc21ofeGdZVc5VCJUJWGtvSFqvh7ABeAv0NE/58VoBVi/rWlvuoPrmOIis3ye5NHfVA+5hsY+B46wXUPUR7aA8jdXiD78+baadtdx1DuRGPdR9tg9ilurj8sSS8TmNfcOKXLdQhRKViwTgPHvp9uV2CR65YCtleMzm+pq3l5uBfi8IaGrD6x47Oe+Nep4mrw//YT7RGVu1T1znhD6F3XMUSlIBxPnWNEf6iQxa5biPLAZhX56aGyfau5sql4hePbjhUxGwB83nVLAXoH1p4bb5i6zXUIUSn5cDuXVg4Bh6RPVRYmItW/y8XFOACkYYu09FTDZL4L4FoAk1z3EI0mAXoVuMvLBH7Ot8lEboTj6fli9AYoFgEwrnuIRpMAL1rVm3vLQw89dZ5kXffQyAvHUyeJ4GkAx7puKSA7PZH5zfXVr7sOISpFi9bs+ouxfX2/BfAl1y0FREX16pZIzapcXZADQMqZaGxTWbZs0qWiegOAWa57iEbYb1XlF4Hs7rbmxpn9rmOICFjS9l7IU2+ZqHwHPDmYilsWwIOwuC3eEHrJdQyNvmh71wzft48D+KzrlgKw2xezoK2+aqPrEKJS9uEQ8AkAp7tuKQB9AlzaEg615vKiHABSzjU1qXl1Troegu8KMB/8PaPi0QcgBmvu4Im+RPmrtiM1LpjVywD5FoAvuu4hyqHdInjAWtyViIQ2u44htzgEHJQdYrC4pS70gusQIgJqO1KTg1lpB/Qs1y15S3BIrV6aiNS05f7SRCOo4aGtJ6jnXQFgOYBqxzlEQ/WOit4V6Dermhure1zHENHgRVq6zlRjvy1AFMAY1z1EQ6CArhPgngnv9yfuu2LGIddBlD+ise4qG8w+psBs1y35RoE3Ap5Z1LykKum6hYj+JBrbOtYPeL+CIOy6JQ/thZG6eF31upG4OAeANCoWrNPAsXtSi9SaqyF6MYCA6yaiwxIcUkXCiK6a/UroyaYmsa6TiGjoorGtk/yguRyQKwDMdd1D9OkkBdX7rHirWsOVW1zXUP6KxnYfY4OHVikQcd2SRzao2iWJyNRdrkOI6OOamtRsPDV9qyquc92SLxR4A2qWJiJVr43UPTgApFEXaempVq//r0XlSvAEM8ozAryoIqv6Av0PPrz4uF7XPUSUew1t2+eqNVcAejl4eBXllywgj4jY/9h1TOgRHupBg6YqDYn091TwUwBB1zlOKe7xsv51zY3TDrpOIaLDC8dTfyOCWwCMc93ikgiaMxK8qr2uYu+I3mckL050WKoSaU+fCx9XQRABcLTrJCpZ3Qo8aIy/qqVuWqfrGCIaHYvWvDVmbP+4ekCWQ/FVAJ7rJipRKp0Q/XXWZP+zve4zKdc5VLjC8dQ5AvwaghrXLQ7shOCaeH0o4TqEiAav4aHUF+DhAQVOc93iQ
```

Comment:
would not do this here as this is command specific, right now you are adding it to all commands